src/kubectl-object-model.ts

```diff
@@ -65,6 +65,14 @@ export class Kubectl {
         return this.execute(args);
     }
 
+    public labelFiles(files: string | string[], labels: string[], overwrite?: boolean): IExecSyncResult {
+        let args = ['label'];
+        args = args.concat(['-f', this.createInlineArray(files)]);
+        args = args.concat(labels);
+        if (!!overwrite) { args.push(`--overwrite`); }

```

Comment:
Is there a scenario which demands a non overwrite scenario?

Reply:
There is no scenario currently with no overwrite. Kept it aligning to `annotate` method, for future scenarios. Want me to remove it?

Reply:
Yep.